test/archethic/beacon_chain/update_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule Archethic.BeaconChain.UpdateTest do
   alias Archethic.P2P.Node
 
   import Mox
-  import GenServer
+  alias GenServer
```

Comment:
I don't see the need of the alias or the import
Also you should use wrapped functions in the module instead (`subscribe`, ...)

Reply:
multiple Update processes(are spawned) with send and assert_recieve , 
maybe due to that we require specific pid GenServer cast

Reply:
I don't think it's the reason, but better is to abstract genserver calls through the API even for tests